packages/cashier-v2/src/lib/DepositCrypto/DepositCrypto.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Divider } from '@deriv-com/ui';
+import DepositCryptoAddress from './components/DepositCryptoAddress/DepositCryptoAddress';
+import DepositCryptoCurrencyDetails from './components/DepositCryptoCurrencyDetails/DepositCryptoCurrencyDetails';
+import DepositCryptoDisclaimers from './components/DepositCryptoDisclaimers/DepositCryptoDisclaimers';
+import DepositCryptoTryFiatOnRamp from './components/DepositCryptoTryFiatOnRamp/DepositCryptoTryFiatOnRamp';
+import styles from './DepositCrypto.module.scss';
```

Comment:
Nit: Maybe add index.ts to import from the components folder.

Reply:
sorted